git ignore patch

Copyright (C) 2013 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

Ignore some files generated at build time.

diff --git a/.gitignore b/.gitignore
index d91e91b..ca8efac 100644
--- a/.gitignore
+++ b/.gitignore
@@ -77,3 +77,5 @@ cscope.*
 
 # spl ais files
 /spl/*.ais
+
+include/configs/onie_version.h
